plumber/ppl/lib/ppl/definition_reviser/blocks_reviser.ex

```diff
@@ -113,7 +115,7 @@ defmodule Ppl.DefinitionReviser.BlocksReviser do
     with {:ok, ppl}        <- PplsQueries.get_by_id(ppl_req.id),
          {:ok, promoter}   <- promoted_by?(ppl_req.request_args),
          {:ok, triggerer}  <- triggered_by?(ppl_req),
-    do: set_ppl_env_vars_(block_def, ppl_req, ppl, promoter, triggerer)
+    do: set_ppl_env_vars_(block_def, ppl_req, ppl, promoter, triggerer, block_def)
```

Comment:
The `block_def` is already the first parameter of the `set_ppl_env_vars_` function.

Reply:
👍🏻